41BullAndCows/css/style.css

```diff
@@ -137,9 +138,15 @@ hr {
     max-height: 50vh;
     overflow: auto;
     font-size: 0.8rem;
+    text-align: center;
 }
 
 p:first-child {
-    color: orangered;
+    color: orange;
```

Comment:
best way would be to create a common color code file and name it as theme.js and import color code from it.

`
primary001: "orange"
white: "000";
black: "fff;
`